public/app/features/templating/template_srv.ts

```diff
@@ -276,6 +279,17 @@ export class TemplateSrv implements BaseTemplateSrv {
   }
 
   replace(target?: string, scopedVars?: ScopedVars, format?: string | Function): string {
+    if (scopedVars && scopedVars.__sceneObject && scopedVars.__sceneObject.value instanceof SceneObjectBase) {
+      const { __sceneObject, ...otherScopedVars } = scopedVars;
+
+      return sceneGraph.interpolate(
+        scopedVars.__sceneObject.value,
+        target,
+        otherScopedVars,
+        format as string | CustomFormatterFn | undefined
+      );
+    }
```

Comment:
```suggestion
    if (scopedVars && scopedVars.__sceneObject && scopedVars.__sceneObject.value instanceof SceneObjectBase) {
      return sceneGraph.interpolate(
        scopedVars.__sceneObject.value,
        target,
        scopedVars,
        format as string | CustomFormatterFn | undefined
      );
    }
```

don't think we need to remove the __sceneObject from scopedVars, just adds unnecessary object creation, At least I don't think it will do any harm reminaing part of the scopedVars object we pass to sceneGraph.interpolate